apps/chronicles/mad.config.ts

```diff
@@ -31,7 +32,7 @@ export const config: MadConfig = {
         whatever: "",
     },
     about: {
-        endpoints: [],
-        buildNumber: "",
+        endpoints: ["https://api.statoil.com/app/mad/test/api/v1"],
+        buildNumber: getBuildNumber(),
```

Comment:
`react-native-device-info` has a method `getBuildNumber()` which picks it up from the app itself. This would let us get the build number in the About Screen without sending it in the mad-config. The only downside is it doesn't give you a value for web. Let's have a discussion on which approach we prefer @HaakonSvane @dansvend @HavardNot

Reply:
I think we should aim for a platform agnostic solution. The current implementation is also config file type agnostic making it possible to use `env` or `json` files.

Reply:
Lets go for that solution!